volumina/volumeEditor.py

```diff
@@ -142,7 +142,7 @@ def lastImageViewFocus(self, axis):
         self._lastImageViewFocus = axis
         self.newImageView2DFocus.emit()
 
-    def __init__( self, layerStackModel, parent, labelsink=None, crosshair=True, syncAlongAxes=(0,1)):
+    def __init__( self, layerStackModel, parent, labelsink=None, crosshair=True, syncAlongAxes=(0,1), editable=True):
```

Comment:
Same as above, I'd rather have a method like "setEditable(flag)" than change a generic method signature for just one applet use case.